com.unity.renderstreaming/Editor/PropertyDrawers/SignalingSettingsDrawer.cs

```diff
@@ -27,18 +27,34 @@ public override VisualElement CreatePropertyGUI(SerializedProperty property)
             return root;
         }
 
-        ISignalingSettingEditor CreateEditor(SerializedProperty property)
+        static ISignalingSettingEditor CreateEditor(SerializedProperty property)
         {
-            var handler = property.serializedObject.targetObject as SignalingManager;
-            var settings = handler.GetSignalingSettings();
+            SignalingSettings settings = null;
```

Comment:
Please fix this like below.
https://github.com/Unity-Technologies/UnityRenderStreaming/commit/4423dda8a8923dac4e531fea2cb56e1568ff2d74